.github/ISSUE_TEMPLATE/investigation.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Investigation 
+labels: 'investigation'
+about: Discuss a broad topic or concern in openssl 
```

Comment:
To me, `investigation` and `discussion` are two very different things.

If this is for another class of GitHub issue, would `Indicate a broad topic or concern in OpenSSL` (note: not `openssl`) be better?

To some degree, calling them investigations might be the thing that's not right, but Discussion is already a thing in and of itself.

Reply:
Yeah, our term namespace is a bit polluted in this effort.  

My intention here is somewhat selfish.  I've identified a number of issues that we have open, which state a general concern which the development community needs space to discuss/investigate/generally talk about, but from a planning purpose is not useful.

https://github.com/openssl/openssl/issues/14837 was the issue that inspired this

@DDvO opened an issue that was very general in nature, sparked a good amount of conversation, and resulted in several improvements, but the issue itself, being wide ranging and without express action identified, is not something I can plan on the roadmap.  It does still however have some validity in that there is debate about the potential for more to be done, and the participants have expressed interest in keeping the issue open to facilitate further conversation

So to recognize that need, and the benefit of these issues, while allowing improved planning visibility, I proposed this: To mark such wide ranging issues that have no direct expected output, allowing them to remain active, while at the same time allowing them to be ignored from a planning perspective (planning being the process of identifying concrete actions to take in the modification of the code base, and scheduling them for completion).

So the name is potentially not perfect, but a rose by any other name....

I'm happy to consider alternative names, but for want of a label, this works for me, especially if we have a template that describes its purpose.

Reply:
Yes, the intent seems clear, and worthwhile, it was the language that was bugging me.

How would you feel about changing

```
Discuss a broad topic or concern in OpenSSL
```

to

```
Raise a broad topic or concern in OpenSSL
```

I can't think of a better term than "investigation" - except possibly "investigation needed" :) - maybe it makes sense to add that to the about string?

```
Raise a broad topic or concern in OpenSSL where more investigation is needed
```

I'm happy with whatever you'd like to do at this point

Reply:
I'm fine with both of those, though Id like to shorten:
```
Raise a broad topic or concern in OpenSSL where more investigation is needed
```
to
```
Raise a broad topic or concern in OpenSSL requiring investigation
```
As the latter keeps the description field under 80 chars :)

Reply:
Looks good - nice